tests/mypy_test.py

```diff
@@ -480,6 +483,18 @@ def test_third_party_stubs(code: int, args: TestConfig, tempdir: Path) -> TestRe
         if spec_matches_path(gitignore_spec, distribution_path):
             continue
 
+        metadata = read_metadata(distribution)
+        if not metadata.requires_python.contains(PYTHON_VERSION):
+            msg = f"skipping {distribution!r} on Python {PYTHON_VERSION} (requires Python {metadata.requires_python})"
```

Comment:
Maybe make it slightly clearer that the issue is the Python version the test is being _run_ with, not the Python version passed on the command line:

```suggestion
            msg = f"skipping {distribution!r} (requires Python {metadata.requires_python}; test is being run using Python {PYTHON_VERSION})"
```